app/src/components/modal/modal_deposit_withdraw/index.tsx

```diff
@@ -271,6 +271,10 @@ export const ModalDepositWithdraw = (props: Props) => {
         await XdaiService.waitForBridgeMessageStatus(hash, context.library)
       }
 
+      if (exchangeType === ExchangeType.withdraw && currencySelected === ExchangeCurrency.Omen) {
+        await XdaiService.waitForClaimSignature(hash, context.library)
+      }
+
```

Comment:
this will need to be change to exchangeType === ExchangeType.withdraw && currencySelected !== ExchangeCurrency.Dai. Because of changes introduced with token list pr so it supports all omni currencies